assets/product.js

```diff
@@ -107,30 +99,32 @@ function uploadImage(element) {
 }
 
 (function productImageHoverZoomer() {
-  const singleProductImagesPreview = document.querySelectorAll(
-    '.product-images-container'
-  );
+  const singleProductImagesPreview = document.querySelectorAll('.product-images');
 
   if (!singleProductImagesPreview || !singleProductImagesPreview.length) return;
 
   singleProductImagesPreview.forEach((imagesPreview) => {
-    const imgZoomer = imagesPreview.querySelector('#img-zoomer-box');
+    const imgZoomers = imagesPreview.querySelectorAll('.img-zoomer-box');
```

Comment:
why you query here all `.img-zoomer-box`? you already have the parent section container

Reply:
I have changed the logic a little bit based on the new theme now the` img-zoomer-box ` will be on each image